4.4.8/Dockerfile.datahub-plugin

```diff
@@ -0,0 +1,21 @@
+# Use the official GeoNetwork v4.4.8 image as the base
+FROM geonetwork:4.4.8
+
+ARG GEONETWORK_VERSION=4.4.8
```

Comment:
The base image uses `GN_VERSION` so defining this new variable is probably not needed